test/FunctionalTests/Microsoft.OData.Core.Tests/ODataMessageWriterTests.cs

```diff
@@ -1030,6 +1078,32 @@ private static IEdmModel GetEdmModel()
             return edmModel;
         }
 
+
+        private static IEdmModel GetLegacyScaleAndSridModel()

```

Comment:
Since this model is used for both the legacy and the new versions:
```suggestion
        private static IEdmModel GetScaleAndSridModel()

```